### PR TITLE
Updating Session :token, :refresh_token and :expires_in after refresh access_token.

### PR DESCRIPTION
**Pull request by alexandrebini**
## What?

Updating Session :token, :refresh_token and :expires_in after refresh access_token.

## Why?

It was just refreshing the ```access_token``` instance, so ```session.token``` does not stay in sync with ```session.access_token.token```

refresh

### CODE REVIEW

**Comment by botdylan** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/5461c4f6e30ba075bc0a0ab0/compare/fix/updating-token-refresh_token-and-expires_in-after-refresh">analyzed this pull request</a>.

**Comment by alexandrebini** [comment]
@bpedro shoud I bump the version again?

**Comment by csaura** [comment]
:+1:

**Comment by andresgutgon** [comment]
:+1:

File: spec/redbooth-ruby/session_spec.rb [hunk 1]
```diff
@@ -74,12 +74,17 @@
...
+    end
 
-    it 'refreshes the access token' do
-      session.refresh_access_token!
-      expect(access_token.token).to eql('_your_new_user_token_')
-      expect(access_token.refresh_token).to eql('_your_new_user_refresh_token_')
+    context 'refreshes the access_token' do
+      before { session.refresh_access_token! }
+      it { expect(session.access_token.token).to eql('_your_new_user_token_') }
+      it { expect(session.access_token.refresh_token).to eql('_your_new_user_refresh_token_') }
```
Comment (masylum) [inline spec/redbooth-ruby/session_spec.rb]: way to go!

**Pull request merged**